app/presenters/show_presenter.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ShowPresenter < Arclight::ShowPresenter
+  def heading
+    "Guide to the " + document.normalized_title
```

Comment:
Will this clobber the environment tag at the beginning of the tab title in the lower envs?

Reply:
Nope, see attached image.
![image](https://github.com/nla/nla-arclight/assets/153031672/d7ba2f5a-973c-4f3d-8e90-68c16b7ffd1b)